lib/tasks/vacations.rake

```diff
@@ -7,4 +7,26 @@ namespace :vacations do
       vacation.save(validate: false)
     end
   end
+
+  desc "Send a e-mail as remember with the pending vacations to be approve"
+  task alert_pending_approve: :environment do
+    Rails.logger.info "anything"
+    vacations = Vacation.pending
+
+    hr_pending_approves = vacations.where(hr_approver_id: nil)
+    if hr_pending_approves.any?
+      Rails.logger.info "Remembering the HR of pending vacations"
+      User.by_roles_in([:hr]).each do |hr_user|
```

Comment:
enumerize provides a scope for each enum option here you can do, User.hr

Reply:
we are using the multiple option there: `:scope option does not work with option :multiple`